RecipesAppTeamPlate/RecipesAppTeamPlate/AppBuilder.swift

```diff
@@ -0,0 +1,67 @@
+// AppBuilder.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Протокол билдер
+protocol Builder {
```

Comment:
```suggestion
protocol BuilderProtocol {
```
всем протоколам в конце Protocol